scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/qfn/vqfn.yaml

```diff
@@ -561,6 +561,50 @@ VQFN-32-1EP_5x5mm_P0.5mm_EP3.1x3.1mm:
   #suffix: '_Pad{pad_x:.2f}x{pad_y:.2f}mm_HandSolder'
   #include_suffix_in_3dpath: 'False'
 
+VQFN-32-1EP_5x5mm_P0.5mm_EP3.15x3.15mm:
+  device_type: 'VQFN'
+  size_source: 'https://www.ti.com/lit/ds/slvs589d/slvs589d.pdf#page=33'
+  ipc_class: 'qfn'
+  body_size_x:
+    minimum: 4.85
+    nominal: 5
+    maximum: 5.15
+  body_size_y:
+    minimum: 4.85
+    nominal: 5
+    maximum: 5.15
+  overall_height:
+    minimum: 0.8
+    maximum: 1
+  lead_width:
+    minimum: 0.18
+    maximum: 0.3
+  lead_len:
+    minimum: 0.3
+    maximum: 0.5
+
+  EP_size_x:
+    minimum: 2.95
+    nominal: 3.15
+    maximum: 3.25
+  EP_size_y:
+    minimum: 2.95
+    nominal: 3.15
+    maximum: 3.25
+  EP_num_paste_pads: [2, 2]
+
+  thermal_vias:
+    count: [3, 3]
+    drill: 0.2
+    paste_via_clearance: 0.1
+    EP_paste_coverage: 0.7
+    grid: [1.2, 1.2]
```

Comment:
Why did you choose those values?

Reply:
Sorry I had not updated all the values in this section from `VQFN-32-1EP_5x5mm_P0.5mm_EP3.1x3.1mm`.

The grid values I have updated them to `1.0, 1.0` match the datasheet.

I note that `paste_via_clearance` and `EP_paste_coverage` have default values in the script, so for now I have removed them from this definition entirely. If that is not correct I would appreciate some guidance on how to select the values.

Reply:
Good choice on using defaults for the EP parameters.
I suggest to do the same with the grid.

This footprint is supposed to be build according to IPC spec (it has no `Texas_` prefix). With the current values (grid, thermal_vias, EP_num_paste_pads) you are mixing both. Resulting in vias below the paste which will lead to solder loss.
Without the grid-setting, the vias are placed no-overlapping with the paste.

Reply:
Thanks for the explanation, I have removed the grid line to allow the defaults, as you say the vias now don't overlap with the paste.

I've added a screenshot of the ThermalVias variant to the top of the PR.